fe/fe-core/src/main/java/com/starrocks/system/ComputeNode.java

```diff
@@ -0,0 +1,90 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.system;
+
+import com.starrocks.alter.DecommissionBackendJob;
+import com.starrocks.catalog.Catalog;
+import com.starrocks.common.FeMetaVersion;
+import com.starrocks.common.io.Text;
+import com.starrocks.server.GlobalStateMgr;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.DataInput;
+import java.io.DataOutput;
+import java.io.IOException;
+
+public class ComputeNode extends Backend {
```

Comment:
I think this isn't reasonable.  Backend has the compute and storage ability，ComputeNode only does compute. I think we could add an ICompute interface.  and let ComputeNode and Backend both implement it.

Reply:
maybe it is better that to add an ICompute interface. and let ComputeNode and Backend both implement it.
but if we do this, it needs to add more work, in order to reuse some implementations of backend in the early stage, so now extends backend.

Reply:
I think we no any reason to make the subset to extend the superset. I evaluated the change, we only need to change th e place that BE only  express the compute ability, the change is acceptable.